packages/now-python/now_init.py

```diff
@@ -1,11 +1,14 @@
-
+import os
 import sys
 import base64
 import json
 import inspect
 from importlib import util
 from http.server import BaseHTTPRequestHandler
 
+# Add the Python packages directory to the search path.
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), '__NOW_PACKAGES_DIR'))
```

Comment:
Does this work with python 3.6, 3.7, and 3.8?

Reply:
It does (also confirmed with a small test).